layers/meta-resin-genericx86/recipes-support/hostapp-update-hooks/files/999-resin-boot-cleaner

```diff
@@ -25,4 +25,10 @@ if [ "$DURING_UPDATE" = "1" ]; then
 			fi
 		fi
 	fi
+
+	# make sure the bootstrap code (boot.img) is removed in case we are using EFI boot
+	if [ -d /sys/firmware/efi ] ; then
+		device="/dev/"$(findmnt --noheadings --canonicalize --output SOURCE /mnt/boot/ | xargs lsblk -no pkname)
```

Comment:
The section above this code uses "$RESIN_BOOT_MOUNTPOINT", is that the same as `/mnt/boot`, would that be better?